homeassistant/components/honeywell/climate.py

```diff
@@ -63,7 +60,6 @@
         vol.Optional(
             CONF_HEAT_AWAY_TEMPERATURE, default=DEFAULT_HEAT_AWAY_TEMPERATURE
         ): vol.Coerce(int),
-        vol.Optional(CONF_REGION, default=DEFAULT_REGION): vol.In(REGIONS),
```

Comment:
Now I see that the default region is EU. So all users, ie all users are US, will be affected by this invalidation. I think we should keep the option in the schema but also mark it as `cv.deprecated`, and just ignore it in the setup code. Remove the default EU region.